examples/tests/tests.ts

```diff
@@ -4,11 +4,29 @@ const loadViaRequire = createRequire(import.meta.url);
 // import test from 'tape';
```

Comment:
I think I would usually expect the tests to be either in top-level `test/` directory or in test files next to the files under test (`src/my_actual_file_test.ts`).

Reply:
I see, I'll address this along with a more comprehensive set of unit tests in a new PR today